src/test/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImportIntegrationTest.java

```diff
@@ -872,6 +880,64 @@ public void testYouCantWriteIntoAnExistingDirectory(){
         writeToGenomicsDB(LOCAL_GVCFS, INTERVAL, workspace, 0, false, 0, 1);
     }
 
+    @Test(expectedExceptions = CommandLineException.class)
+    public void testOverwriteWorkspaceAndIncrementalImportCannotBothBeTrue() {
+        final String workspace = createTempDir("genomicsdb-incremental-tests").getAbsolutePath() + "/workspace";
+        writeToGenomicsDB(LOCAL_GVCFS, INTERVAL, workspace, 0, false, 0, 1, false, true, true);
+    }
+
+    @Test(expectedExceptions = UserException.class)
+    public void testIncrementalMustHaveExistingWorkspace() {
+        final String workspace = createTempDir("genomicsdb-incremental-tests").getAbsolutePath();
+        writeToGenomicsDB(LOCAL_GVCFS, INTERVAL, workspace + "workspace2", 0, false, 0, 1, false, false, true);
+    }
+
+    @Test
+    public void testGenomicsDBBasicIncremental() throws IOException {
+        final String workspace = createTempDir("genomicsdb-incremental-tests").getAbsolutePath() + "/workspace";
+        writeToGenomicsDB(LOCAL_GVCFS.subList(0,2), INTERVAL, workspace, 0, false, 0, 1, false, false, false);
+        checkJSONFilesAreWritten(workspace);
+        writeToGenomicsDB(LOCAL_GVCFS.subList(2,3), INTERVAL, workspace, 0, false, 0, 1, false, false, true);
+        checkJSONFilesAreWritten(workspace);
+
+        checkGenomicsDBAgainstExpected(workspace, INTERVAL, COMBINED_WITH_GENOTYPES, b38_reference_20_21, true, ATTRIBUTES_TO_IGNORE, true, false);
```

Comment:
I would strongly prefer that this be a comparison to CombineGVCFs behavior or compare importing one sample and then the other to a single import with both samples.  The goal is for this functionality to be equivalent, so I'd rather show that it is than rely on expected outputs.

Reply:
So, I'm actually piggybacking on an existing expected output instead of introducing a new one. As such, I think this should be checking something like the latter (import all samples at once and comparing against expected output, then break up into  multiple sets of samples that get incrementally imported and compare against the same expected output). The "import all samples at once" part is checked via this existing test:

```
@Test
public void testGenomicsDBThreeLargeSamplesWithGenotypes() throws IOException {
    ArrayList<SimpleInterval> intervals = new ArrayList<SimpleInterval>(Arrays.asList(new SimpleInterval("chr20", 1, 64444167)));
    testGenomicsDBImporterWithGenotypes(LOCAL_GVCFS, intervals, COMBINED_WITH_GENOTYPES, b38_reference_20_21, true, true, false);
}
```
And then the incremental test checks the "break up into  multiple sets of samples that get incrementally imported" part. Am I missing something?

I can certainly change it to have the incremental tests compare two workspaces instead of comparing against expected output if you prefer. I also noticed that I only have cases that test importing a single interval...will change some to multiple intervals.

Reply:
Got it -- it wasn't obvious at first glance that the expected output was also used in another test.